integration/client-cli/admin-cli/src/main/java/org/keycloak/client/admin/cli/commands/AbstractRequestCmd.java

```diff
@@ -440,4 +439,14 @@ private boolean isUpdate() {
     private boolean isCreateOrUpdate() {
         return "post".equals(httpVerb) || "put".equals(httpVerb);
     }
+
+    private boolean unGetByID(String url) {
```

Comment:
This would read better as isGetByID.  

Also is it possible to add a test for this?

Reply:
Sorry, I can't add UT for this.